connectors/http/src/main/java/quarks/connectors/http/HttpStreams.java

```diff
@@ -38,12 +40,88 @@ Licensed to the Apache Software Foundation (ASF) under one
  */
 public class HttpStreams {
     
+    /**
+     * Make an HTTP GET request with JsonObject. <br>
+     * 
+     * Method specifically works with JsonObjects. For each JsonObject in the stream, 
+     * HTTP GET request is executed on provided uri. As a result, Response is added to
+     * the response TStream.
+     * <br>
+     * 
+     * Sample usage:<br>
+     * 
+     * <pre>
+     * {@code
+     *     JsonObject request1 = new JsonObject();
+     *     request1.addProperty("a", "abc");
+     *     request1.addProperty("b", "42");
+     *     TStream<JsonObject> rc = HttpStreams.getJson(
+     *             topology.collection(Arrays.asList(request1)),
```

Comment:
nit: move the construction of the input stream outside of the getJson() invocation?  It's not something that will ever occur in practice.  Ditto the postJson() example below.